app/src/main/res/layout/item_description_fragment.xml

```diff
@@ -1,18 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<de.danoeh.antennapod.view.VerticalNestedScrollableHost xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There is a problem with this (that's why I initially wrote that handling scrolling is going to be hard). When you are trying to scroll up a long description in a 45° angle, it goes to the cover instead of scrolling up the WebView.

Also, the scroll bar of the WebView acts a bit strange. It goes below the playback controls, which makes it look like the WebView continues below the controls.

Reply:
Yea noticed that to, but it doesnt happen a lot, maybe this could be optimized later on...

Didnt noticed the scrollbar until now, we could hide it?

Reply:
> maybe this could be optimized later on...

No, sorry. It already was annoying the few minutes I played around with it. We already break the muscle memory of users with this change. When it then also has glitches, I don't think they will be delighted.

> Didnt noticed the scrollbar until now, we could hide it?

Hmm. That just hides the problem instead of solving it. I think the scrollbar is useful and should be kept.

Reply:
Ok made a fix, 45° issue is gone and I added the NestedScrolHost back to the FeedItem

For the scrollbar I don't know why this happens, it's this because of the webview?

Reply:
The issue is still happening for me. See the videos here:

Develop branch: https://drive.google.com/file/d/1xE_xGQYkWSPaRFacquN0okWfw0_0W5We/view?usp=sharing
Your branch: https://drive.google.com/file/d/1G3vdvryKbY-f14mKXr2__7P6Xdh00pF2/view?usp=sharing

Yours does not "fling" when touched. I believe that removing the inner ViewPager could help, or at least make it easier to deal with the problem. If you want to try the same episode, you can look at the episode "Table shownotes" in this feed: https://tools.bytehamster.com/podcast/rss.xml

Reply:
I think this is unrelated to the viewpager in viewpager, this is due to NestedScrollHost and how it calculates when to intercept.

Reply:
Ah thanks, great rss!

The webview scrollbar issue is due to `ShownotesWebView` `onMeasure` function, if removed the scrollbars correct, is this function requiered?

Reply:
Ok i see your point with the, so it will be hard to account for both viewpagers if they go in different direction so removing the tablayout/viepager seems like a sensible idea, an alternative could be a fake tablayout () that just switches the fragments (description/transcript) without a viewpager